web/src/SecureVote/SPAs/SwarmMVP/Model.elm

```diff
@@ -11,7 +11,7 @@ import SecureVote.SPAs.SwarmMVP.Routes exposing (DialogRoute(NotFoundDialog), Ro
 
 type alias Model =
     { mdl : Material.Model
-    , snack : Material.Snackbar.Model Msg
+    , snack : Material.Snackbar.Model String
```

Comment:
This is type param is what the Snackbar _returns_, not what it accepts. Best to keep it as Msg

Reply:
I can't get it to work with `Msg`. It seems to correspond to the payload of the toast http://package.elm-lang.org/packages/debois/elm-mdl/8.1.0/Material-Snackbar#toast

And I'm just using an essentially empty string as the payload
https://github.com/swarmfund/sv-voting-mvp/pull/93/files#diff-0477ecf9d9cbc18baca0520e19a6159aR83

The demo uses 
```
type alias Model =
  { count : Int
  , snackbar : Snackbar.Model Int
  ...
  }
```
Because the payload in the Toast is model.count
https://github.com/debois/elm-mdl/blob/master/demo/Demo/Snackbar.elm#L42